apps/E2E/src/common/BasePage.ts

```diff
@@ -104,6 +124,15 @@ export class BasePage {
     await (await this._primaryComponent).click();
   }
 
+  /* The goal of click() and sendKeys() is to be generally used across all pageobjects to reduce code repetition in similar methods. */
+  async click(element: Promise<WebdriverIO.Element>): Promise<void> {
+    await (await element).click();
+  }
+
+  async sendKeys(element: Promise<WebdriverIO.Element>, keys: Keys[]): Promise<void> {
```

Comment:
Do we need `: Promise<void>`?

Reply:
It isn't necessary, but is there anything wrong with explicitly stating the return of both functions in this case?